task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -47,13 +47,19 @@ spec:
           memory: 512Mi
           cpu: 10m
       script: |
-        #!/usr/bin/env bash
+                #!/usr/bin/env bash
+
         set -o pipefail
+
         source /utils.sh
 
-        declare -a ALLOWED_BASE_IMAGES=("registry.redhat.io/openshift4/ose-operator-registry")
+
+        declare -a
+        ALLOWED_BASE_IMAGES=("registry.redhat.io/openshift4/ose-operator-registry")
+
 
         ### FBC base image check
+
         if [ -z "${BASE_IMAGE}" ]; then
```

Comment:
Why all those style changes?
I'm not even sure if `declare -a` would work now

Reply:
This was unintentional.  I was doing some editing in the openshift yaml editor when I was testing the task.  It seems to have done some weird things with the formatting.  I'll go back through and fix things.